modules/cardano.nix

```diff
@@ -101,7 +101,9 @@ wallet:
       publicIP = if options.networking.publicIPv4.isDefined then config.networking.publicIPv4 else null;
       privateIP = if options.networking.privateIPv4.isDefined then config.networking.privateIPv4 else "0.0.0.0";
       statsdServer = "127.0.0.1:8125";
-    };
+    }) // (optionalAttrs params.typeIsCore (let assetPath = ../static/asset-locked-addresses.nix; in {
```

Comment:
Would it be better to fail hard if `static/asset-locked-addresses.nix` is not present and we are in the production environment? Trying to prevent the case of where the file is accidentally named wrong or something like that.